assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -7063,6 +7063,8 @@ private void post_save_submission(RunData data, boolean post) {
      * Takes the inline submission, prepares it as an attachment to the submission and queues the attachment with the content review service
      */
     private void prepareInlineForContentReview(String text, AssignmentSubmission submission, SessionState state, User student) {
+
+        System.out.println("HERE1");
```

Comment:
Please, remove this. Thanks!

Reply:
good spot, thanks. From another ticket :)